packages/cli/src/commands/inspect.ts

```diff
@@ -98,12 +135,22 @@ export default async function main(client: Client) {
     `Fetching deployment "${deploymentIdOrHost}" in ${chalk.bold(contextName)}`
   );
 
+  const until = Date.now() + timeout;
+  const wait = argv['--wait'];
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Do we want to add another flag to remove the typical verbose output of `vc inspect` that can be paired with `--wait`?

Even on a small project, if I want to use `vc inspect --wait`, I'll get all this output:

```
$  vc inspect --wait https://nextjs-19cgjb33n-uncurated-tests.vercel.app
Vercel CLI 28.18.5-1b347fb
> Fetched deployment "nextjs-19cgjb33n-uncurated-tests.vercel.app" in endangeredmassa [902ms]

  General

    id		dpl_EwDbkYnm35e6ZC2wsPDtGwMNAKBf
    name	nextjs
    status	● Ready
    url		https://nextjs-19cgjb33n-uncurated-tests.vercel.app
    created	Mon Apr 17 2023 10:30:43 GMT-0500 (Central Daylight Time) [32s ago]


  Aliases

    ╶ https://nextjs-endangeredmassa-uncurated-tests.vercel.app


  Builds

    ┌ .        [0ms]
    ├── λ api/[pid] (2.86MB) [sfo1]
    ├── λ api/demo (2.86MB) [sfo1]
    ├── λ _next/data/ZYi_oIKyufen_B3Pcv_4v/blog-fallback-blocking/[id].json (2.87MB) [sfo1]
    ├── λ blog-fallback-blocking/[id] (2.87MB) [sfo1]
    ├── λ _next/data/ZYi_oIKyufen_B3Pcv_4v/blog-fallback-false/[id].json (2.87MB) [sfo1]
    └── 5 output items hidden


  Routes

    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/blog\-fallback\-blocking/(?<id>[^/]+?)(?:/)?.json$              ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/blog-fallback-blocking/[id].json?id=$id
    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/blog\-fallback\-false/(?<id>[^/]+?)(?:/)?.json$                 ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/blog-fallback-false/[id].json?id=$id
    ╶ ^/$                                                                                                ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/index.json
    ╶ ^/$                                                                                                ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/index.json
    ╶ ^/((?!_next/)(?:.*[^/]|.*))/?$                                                                     ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/$1.json
    ╶ ^/((?!_next/)(?:.*[^/]|.*))/?$                                                                     ->      /_next/data/ZYi_oIKyufen_B3Pcv_4v/$1.json
    ╶ ^/blog\-fallback\-blocking/(?<id>[^/]+?)(?:/)?$                                                    ->      /blog-fallback-blocking/[id]?id=$id
    ╶ ^/blog\-fallback\-false/(?<id>[^/]+?)(?:/)?$                                                       ->      /blog-fallback-false/[id]?id=$id
    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/(.*).json$                                                      ->      __next_data_catchall
    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/api/(?<pid>[^/]+?)(?:/)?.json$                                  ->      /api/[pid]?pid=$pid
    ╶ ^/api/(?<pid>[^/]+?)(?:/)?$                                                                        ->      /api/[pid]?pid=$pid
    ╶ ^/_next/__private/trace$                                                                           ->      /404      [404]
    ╶ ^/.*$                                                                                              ->      /404      [404]
    ╶ ^/.*$                                                                                              ->      /500      [500]
    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/(.*).json$                                                      ->      /$1
    ╶ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/(.*).json$                                                      ->      /$1
    ╶ ^/_next/static/(?:[^/]+/pages|pages|chunks|runtime|css|image|media)/.+$                            ->      $0      [404]
    ╶ ^/index(?:/)?$                                                                                     ->      /
    ╶ ^/index(?:/)?$                                                                                     ->      /
    ┌ ^/_next/static/(?:[^/]+/pages|pages|chunks|runtime|css|image|media|ZYi_oIKyufen_B3Pcv_4v)/.+$
    └── cache-control: public,max-age=31536000,immutable
    ╶ ^(?:/(_next/data/[^/]{1,}))?/templates(.*)(.json)?[/#\?]?$
    ╶ ^(?:/(_next/data/[^/]{1,}))?/dashboard(.json)?[/#\?]?$
    ┌ ^/_next/data/ZYi_oIKyufen_B3Pcv_4v/(.*).json$
    └── x-nextjs-matched-path: /$1
    ┌ ^(?:/((?:[^/]+?)(?:/(?:[^/]+?))*))/$                                                                                                                                                      [308]
    └── Location: /$1
    ┌ ^/((?!index$).*)$
    └── x-matched-path: /$1
    ╶ ^/404/?$                                                                                                                                                                                  [404]
    ┌ ^/index$
    └── x-matched-path: /
    ╶ ^/500$                                                                                                                                                                                    [500]
    ╶ ^/.*$                                                                                                                                                                                     [404]
    ╶ filesystem╶ resource╶ miss╶ rewrite╶ hit╶ error
```

When maybe we could have `--basic` (or a much better name):

```
$  vc inspect --wait --basic https://nextjs-19cgjb33n-uncurated-tests.vercel.app/
Vercel CLI 28.18.5-1b347fb
> READY
```

which could pipe just the status output "READY", when successful.